### PR TITLE
persistent ansible changes

### PR DESCRIPTION
**Pull request by shweta83**
Close loop: https://bugzilla.redhat.com/show_bug.cgi?id=1786358
Ability to make persistent changes in "ansible.cfg" on Satellite Server.

### CODE REVIEW

**Review by Griffin-Sullivan — APPROVED** [review]

File: tests/foreman/cli/test_ansible.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    :expectedresults: Changes in ansible.cfg are persistent after running
+        "satellite-installer".
+    """
+    ansible_cfg = "/etc/ansible/ansible.cfg"
+    param = "local_tmp = /tmp"
+    command = f'''echo "{param}" >> {ansible_cfg}'''
+    target_sat.execute(command)
+    assert param in target_sat.execute(f'cat {ansible_cfg}').stdout.strip("\n")
+    target_sat.execute("satellite-installer")
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ansible.py:45]: As this test runs the installer, can this test be moved to destructive?

File: tests/foreman/cli/test_ansible.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        1. Update value in ansible.cfg.
+        2. Verify value is updated in the file.
+        3. Run "satellite-installer".
+        4. Verify the changes are persistent in the file.
+
+    :expectedresults: Changes in ansible.cfg are persistent after running
+        "satellite-installer".
+    """
+    ansible_cfg = "/etc/ansible/ansible.cfg"
+    param = "local_tmp = /tmp"
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ansible.py:41]:
```suggestion
    ansible_cfg = '/etc/ansible/ansible.cfg'
    cfg_param = 'local_tmp = /tmp'
```

File: tests/foreman/cli/test_ansible.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    """Check if changes in ansible.cfg are persistent on running satellite-installer
+
+    :id: c22fcd47-8627-4230-aa1f-7d4fc8517a0e
+
+    :BZ: 1786358
+
+    :customerscenario: true
+
+    :Steps:
+
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ansible.py:31]:
```suggestion
```

File: tests/foreman/cli/test_ansible.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
+
+def test_positive_persistent_ansible_change(target_sat):
+    """Check if changes in ansible.cfg are persistent on running satellite-installer
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ansible.py:22]:
```suggestion
    """Check if changes in ansible.cfg are persistent after running satellite-installer
```

File: tests/foreman/cli/test_ansible.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
+
+def test_positive_persistent_ansible_change(target_sat):
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ansible.py:21]:
```suggestion
def test_positive_satellite_installer_ansible_cfg(target_sat):
```

File: tests/foreman/cli/test_ansible.py (hunk 1 above)
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ansible.py:45]:
```suggestion
    target_sat.execute('satellite-installer')
```

File: tests/foreman/cli/test_ansible.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        4. Verify the changes are persistent in the file.
+
+    :expectedresults: Changes in ansible.cfg are persistent after running
+        "satellite-installer".
+    """
+    ansible_cfg = "/etc/ansible/ansible.cfg"
+    param = "local_tmp = /tmp"
+    command = f'''echo "{param}" >> {ansible_cfg}'''
+    target_sat.execute(command)
+    assert param in target_sat.execute(f'cat {ansible_cfg}').stdout.strip("\n")
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ansible.py:44]:
```suggestion
    assert param in target_sat.execute(f'cat {ansible_cfg}').stdout.splitlines()
```

**Review by Gauravtalreja1 — CHANGES_REQUESTED** [review]

**Review by adarshdubey-star — APPROVED** [review]
Ack, pending comments

File: tests/foreman/cli/test_ansible.py (hunk 4 above)
Comment (shweta83) [reply tests/foreman/cli/test_ansible.py]: Updated.

File: tests/foreman/cli/test_ansible.py (hunk 5 above)
Comment (shweta83) [reply tests/foreman/cli/test_ansible.py]: I renamed it.

File: tests/foreman/cli/test_ansible.py (hunk 1 above)
Comment (shweta83) [reply tests/foreman/cli/test_ansible.py]: Updated.

File: tests/foreman/cli/test_ansible.py (hunk 3 above)
Comment (shweta83) [reply tests/foreman/cli/test_ansible.py]: Updated.

File: tests/foreman/cli/test_ansible.py (hunk 2 above)
Comment (shweta83) [reply tests/foreman/cli/test_ansible.py]: Updated.

File: tests/foreman/cli/test_ansible.py (hunk 1 above)
Comment (shweta83) [reply tests/foreman/cli/test_ansible.py]: Moved to destructive.

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_ansible.py

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**